src/Command/TestCommand.php

```diff
@@ -450,6 +451,7 @@ public function generateFixtureList($subject): array
             $this->_processController($subject);
         }
 
+        /** @psalm-suppress RedundantFunctionCall */
```

Comment:
Had to add that here, otherwise I got
```
ERROR: RedundantFunctionCall - src/Command/TestCommand.php:454:16 - The call to array_values is unnecessary, array<empty, empty> is already a list (see https://psalm.dev/280)
        return array_values($this->_fixtures);
```